src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -57,4 +61,18 @@ public List<String> getAllValues(Prefix prefix) {
     public String getPreamble() {
         return getValue(new Prefix("")).orElse("");
     }
+
+    /**
+     * Throws a {@code ParseException} if any of the prefixes given in {@code prefixes} appeared more than
```

Comment:
For the javadocs, I believe the `@param` and `@throws` tags have to be added
```suggestion
    /**
     * Throws a {@code ParseException} if any of the prefixes given in {@code prefixes} appeared more than
     * once among the arguments.
     *
     * @param prefixes List of prefixes supplied by the user
     * @throws ParseException If duplicate prefixes are present
     */
```

Reply:
@dlimyy Good catch that some tags are missing. Our policy is javadoc tags can be omitted if not adding value e.g., repeating the same information already available elsewhere in the javadoc